### ISSUE TITLE
Yum Repository

### ISSUE THREAD

**Issue by danfoxley** [alerta/alerta#956]
**Is your feature request related to a problem? Please describe.**
Simplify installation and upgrades.  

**Describe the solution you'd like**
An alerta  yum  repository for alerta installation.

**Describe alternatives you've considered**
Docker
Trying to install  python

**Additional context**

**Comment by satterly** [comment]
Alerta is already packaged for easy installation using Python packages. The command to install alerta is...
```
$ pip install alerta-server
```
.. which seems pretty simple to me. And to upgrade ...
```
$ pip install alerta-server --upgrade
```
Which again, doesn't seem overcomplicated.

You say you've tried Docker and "trying to install python" but don't say what commands you ran, what errors or problems you encountered and why having a yum repository would produce a different result.

**Comment by danfoxley** [comment]
I will give pip python a shot and let you know my results. Below look good?

1. I will follow this:
https://github.com/alerta/alerta/#installation

2. which, will only leave question of how to install pip. See any issue with these instructions? (I will plan to follow the EPEL step(s).
https://packaging.python.org/guides/installing-using-linux-tools/#centos-rhel

**Comment by danfoxley** [comment]
I'm giving up on CentOS and will use the tutorial, which uses Ubuntu.
http://docs.alerta.io/en/latest/gettingstarted/tutorial-1-deploy-alerta.html#tutorial-1

Here are the bulk steps I did on CentOS, which I never got to install and stands why the original ask for a feature enhancement for "yum install".  You can see the issues below I ran into, never-mind the leap to get python3 working on CentOS 2.7 system.

Install Python 3.6 per https://packaging.python.org/guides/installing-using-linux-tools/#centos-rhel
sudo yum install https://$(rpm -E '%{?centos:centos}%{!?centos:rhel}%{rhel}').iuscommunity.org/ius-release.rpm
sudo yum install python36u*

Check Python Version
$ python3.6 -V
Python 3.6.7

Quick Tutorial - Not applicable...to obtuse.
http://docs.alerta.io/en/latest/quick-start.html

Deploying an Alerta Server
http://docs.alerta.io/en/latest/gettingstarted/tutorial-1-deploy-alerta.html#tutorial-1

sudo yum install -y mongodb
sudo yum install nginx uwsgi-plugin-python3

Per tutorial
cd /opt
$ python3 -m venv alerta
$ alerta/bin/pip install --upgrade pip wheel alerta-server alerta uwsgi

CentOS Specific (IUS Python specific)
$ sudo python3.6 -m venv alerta


(required running with SUDO)
sudo alerta/bin/pip install --upgrade pip wheel alerta-server alerta uwsgi

--------ERROR FIX (error not shown)
sudo yum install python-psycopg2

--------ERROR FIX (error not shown)
sudo yum install postgresql-9.2.24-1.el7_5.x86_64 

--------ERROR FIX (error not shown)
sudo yum group install "Development Tools"

-------ERROR  - Installation Failure

sudo alerta/bin/pip install psycopg2-binary
ERROR(S)
error: command 'gcc' failed with exit status 1
ERROR: Command "/opt/alerta/bin/python3.6 -u -c 'import setuptools, tokenize;__file__='"'"'/tmp/pip-install-8qbdnkfw/psycopg2/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-r3xqwh9q/install-record.txt --single-version-externally-managed --compile --install-headers /opt/alerta/include/site/python3.6/psycopg2" failed with error code 1 in /tmp/pip-install-8qbdnkfw/psycopg2/

In file included from psycopg/psycopgmodule.c:27:0:
    ./psycopg/psycopg.h:35:22: fatal error: libpq-fe.h: No such file or directory
     #include <libpq-fe.h>
                          ^
    compilation terminated.

**Comment by satterly** [comment]
You're missing header files required to compile the Postgres python package. Postgres python package (`psycopg2`) installation has become a mess because of [this](https://github.com/psycopg/psycopg2/issues/674). Not sure what the easy answer is at this stage but for your situation you need to install the devel package. Try `yum install postgresql-devel` and rerun the `pip install`.

**Comment by satterly** [comment]
Fixed by #967 / #968
